polaris/polaris/templates/polaris/base.html

```diff
@@ -230,5 +233,9 @@
       {% endif %}
     {% endfor %}
 
+    {# Rendered via custom extra_head template #}
```

Comment:
Not sure what this comment means

Reply:
When you inspect the HTML rendered in the browser by this template, the JS scripts at the bottom of the body will be either loaded by the JS script integration function or template extensions. I'd like there to be a visual indicator of which source the scripts came from.

Reply:
But shouldn't it be `extra_body` instead of `extra_head` there?